spring-cloud-alibaba-starters/spring-cloud-starter-stream-rocketmq/src/main/java/com/alibaba/cloud/stream/binder/rocketmq/integration/outbound/RocketMQProduceFactory.java

```diff
@@ -45,12 +49,14 @@
  */
 public final class RocketMQProduceFactory {
 
-	private RocketMQProduceFactory() {
-	}
-
 	private final static Logger log = LoggerFactory
 			.getLogger(RocketMQProduceFactory.class);
 
+	private static Map<String, DefaultMQProducer> PRODUCER_REUSABLE_MAP = new ConcurrentHashMap<>();
```

Comment:
Please add `final` to  PRODUCER_REUSABLE_MAP.

Reply:
fixed